main_OSEM.py

```diff
@@ -63,9 +63,16 @@ def __init__(self, data: Dataset, num_subsets: int = 7, update_objective_interva
 
             acquisition_model_subset = STIR.AcquisitionModelUsingParallelproj()
             acquisition_model_subset.set_additive_term(additive_term_subset)
-            acquisition_model_subset.set_up(prompts_subset, self.x)
+            if prompts_subset.dimensions()[0] > 1 and multiplicative_factors_subset.dimensions()[0] == 1:
+                # non-TOF multiplicative factors for TOF data
```

Comment:
Oh is this a backwards-incompatible change? Current participants won't be aware of this.

Reply:
removed this now.